aci/resource_aci_fvesg.go

```diff
@@ -0,0 +1,696 @@
+package aci
+
+import (
+	"fmt"
+	"github.com/ciscoecosystem/aci-go-client/client"
+	"github.com/ciscoecosystem/aci-go-client/models"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"log"
+)
+
+func resourceAciEndpointSecurityGroup() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAciEndpointSecurityGroupCreate,
+		Update: resourceAciEndpointSecurityGroupUpdate,
+		Read:   resourceAciEndpointSecurityGroupRead,
+		Delete: resourceAciEndpointSecurityGroupDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: resourceAciEndpointSecurityGroupImport,
+		},
+
+		SchemaVersion: 1,
+
+		Schema: AppendBaseAttrSchema(map[string]*schema.Schema{
+			"application_profile_dn": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"annotation": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
```

Comment:
Annotation field is not needed here, because I added annotation in the base attribute schema.